sansa-rdf-common/src/main/scala/net/sansa_stack/rdf/common/partition/utils/RdfTerm.scala

```diff
@@ -11,31 +10,30 @@ import org.apache.jena.graph.Node
 case class RdfTerm(t: Int, v: String, lang: String, dt: String)
 
 object RdfTerm {
-  def toLexicalForm(o: Any) = "" + o //NodeFmtLib.str(node)
+  def toLexicalForm(o: Any): String = "" + o // NodeFmtLib.str(node)
 
-  def termToNode(term: RdfTerm) = {
+  def termToNode(term: RdfTerm): Unit = {
```

Comment:
Is the return type correct? Shouldn't it be `Node` ?

Reply:
Indeed, it should be `Node` instead. Autocomplete did the job:). Will fix it now! 
Thanks for it, Patrick.

Reply:
Done!